pkg/controller/etcdproxy/etcdproxy_controller.go

```diff
@@ -287,6 +287,15 @@ func (c *EtcdProxyController) syncHandler(key string) error {
 		return nil
 	}
 
+	// Deploy Server Etcd Proxy certificates.
+	if err = c.createEtcdProxyCACertConfigMap(etcdstorage); err != nil {
```

Comment:
Ok, maybe it's about names.  This is the proxyClientCABundle used to verify that the aggregated apiserver is authorized to speak to the proxy?